src/Components/BaseGroup.jsx

```diff
@@ -1,50 +1,38 @@
 import React from 'react'
-import IconButton from '@mui/material/IconButton'
-import Tooltip from '@mui/material/Tooltip'
 import {makeStyles} from '@mui/styles'
+import AboutControl from './AboutControl'
 import Settings from './Settings'
-import AboutControl from './AboutPanel'
-import Open from '../assets/Icons/Open.svg'
+import {TooltipIconButton} from './Buttons'
+import OpenIcon from '../assets/2D_Icons/Open.svg'
 
 
 /**
  * Base group contains Settings, ModelUpload, About
- * @param {function} fileOpen opens a file dialog for new model upload
- * @param {Number} offsetTop
- * @return {Object} React component.
+ * @param {Object} itemPanel ItemPanel component
```

Comment:
Thx.  Also, I've remove passing around offsetTop since there's now only 1 dialog object and it has multiple callers; passing that variable around seems overkill.  I don't like not having it as a variable, but going to hardcode for now and look into passing via the theme later.